src/components/shared/Button/ActionButton.tsx

```diff
@@ -4,32 +4,40 @@ import { MessageDescriptor } from 'react-intl';
 import { ActionTransformFnType } from '~utils/actions';
 import { useAsyncFunction, useMounted } from '~hooks';
 import DefaultButton from '~shared/Button';
+import { ActionTypes } from '~redux';
 import { Props as DefaultButtonProps } from './Button';
 
+const getFormAction = (
```

Comment:
Right. now I see that you've added a helper here for the success/error

I'm cool with this, however I'd still like to have the option to pass in an action name that will overwrite this.

Something like this:
```
const submit = submit || getFormAction(actionType, 'SUBMIT');
const error = error || getFormAction(actionType, 'ERROR');
const success = success || getFormAction(actionType, 'SUCCESS');
```